bindings/experimental/distrdf/python/DistRDF/__init__.py

```diff
@@ -84,5 +147,6 @@ def create_distributed_module(parentmodule):
     # Inject top-level functions
     distributed.initialize = initialize
     distributed.create_logger = create_logger
+    distributed.RunGraphs = RunGraphs
```

Comment:
I think what we want is to pythonize the original, C++ `RunGraphs` so that if users call it on a standard RDF that one is called, if they call it on distRDFs the new one is called -- users shouldn't have to think whether they need to call `ROOT.RDF.RunGraphs` or `ROOT.RDF.Experimental.Distributed.RunGraphs`

Reply:
Yes, I can provide a pythonization in this PR as well